js/ui.js

```diff
@@ -13,8 +13,14 @@ function display(data) {
 		if (! target)
 			return
 
-		if (typeof value == 'number')
-			value = String(value.toFixed(2)).replace('.', ',')
+		if (typeof value == 'number') {
+			if (target.hasAttribute('data-round'))
+				value = Math.round(value)
+			else
+				value = value.toFixed(2)
```

Comment:
Or `value = value.toFixed(target.attributes['data-decimals'] || 2)`, with `data-decimals="0"` in HTML.